lib/shelley/test/integration/Main.hs

```diff
@@ -348,13 +370,19 @@ bracketTracer' tr name = bracketTracer (contramap (MsgBracket name) tr)
 -- @CARDANO_NODE_TRACING_MIN_SEVERITY@ environment variable.
 nodeMinSeverityFromEnv :: IO Severity
 nodeMinSeverityFromEnv =
-    minSeverityFromEnv Error "CARDANO_NODE_TRACING_MIN_SEVERITY"
+    minSeverityFromEnv Info "CARDANO_NODE_TRACING_MIN_SEVERITY"
 
--- Allow configuring integration tests and wallet log level with
+-- Allow configuring wallet log level with
 -- @CARDANO_WALLET_TRACING_MIN_SEVERITY@ environment variable.
 walletMinSeverityFromEnv :: IO Severity
 walletMinSeverityFromEnv =
-    minSeverityFromEnv Info "CARDANO_WALLET_TRACING_MIN_SEVERITY"
+    minSeverityFromEnv Critical "CARDANO_WALLET_TRACING_MIN_SEVERITY"
```

Comment:
Is it really sound? Sure we will have less logs but we still needs those logs.

Reply:
This is the severity used for `Stdout`.